tools/rosbag/src/player.cpp

```diff
@@ -385,16 +385,16 @@ void Player::printTime()
 
         if (paused_)
         {
-            printf("\r [PAUSED ]  Bag Time: %13.6f   Duration: %.6f / %.6f               \r", time_publisher_.getTime().toSec(), d.toSec(), bag_length_.toSec());
+            printf("\r [PAUSED ]  Bag Time: %13.6f   Duration: %.6f / %.6f", time_publisher_.getTime().toSec(), d.toSec(), bag_length_.toSec());
```

Comment:
The diff removes the trailing `\r` which it shouldn't. Same below.

Reply:
I removed the second carriage return (`\r`) in order for the active carriage to be placed after the text and not at the beginning.